k8s/namespaces/bsp/bulk-scan-payment-processor/bsp-storage-account-creds-demo.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: bsp-storage-account-creds-demo
```

Comment:
this name and filename should match the one in https://github.com/hmcts/cnp-flux-config/pull/16630/files#diff-f40dcaf10cd822b2c4e349e646c7ab220909ff2e87153dd5e8b449ac61c2bc26R16